test/tune/test_tune.py

```diff
@@ -217,6 +217,22 @@ def simple_func(config):
     logger.info(f"BlendSearch best result: {best_trial.last_result}")
 
 
+def test_run_training_function_return_value():
+    from flaml import tune
+
+    def evaluate_config(config):
+        metric = (round(config['x'])-85000)**2 - config['x']/config['y']
+        return {"metric": metric}
+
+    tune.run(
+        evaluate_config,
```

Comment:
The test failed. Either pass `metric='metric'` to tune.run or return a float in `evaluate_config`

Reply:
I've modified the test function to pass a `training_function` returning a dict and another one returning a scalar.